build_rocm_python3

```diff
@@ -29,6 +29,13 @@ TF_PKG_LOC=/tmp/tensorflow_pkg
 rm -f $TF_PKG_LOC/tensorflow*.whl
 
 yes "" | ROCM_PATH=$ROCM_INSTALL_DIR TF_NEED_ROCM=1 PYTHON_BIN_PATH=/usr/bin/python3 ./configure
+# Explicitly define resource constraints on bazel to avoid overload on rocm-ci
+if [ ! -z $2 ]; then
```

Comment:
Can we just use `-n` here? The idea here is if there is and second argument ("1", "foo", etc) then we restrict?

You could perhaps generalize it to take in the settings as the second argument. For example, if the argument is "60000/35/70" you can parse that out and set the resources accordingly.